register.php

```diff
@@ -41,24 +41,23 @@
 				
 					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Register</p></b>
 					
+					<!-- Form for input fields -->
 					<h3><form method = "post" id= "01_login">
 						
-						<p style = "margin: 2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Username:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw;" type = "text" name = "username" placeholder="Your Username..."/>
+						<input style = "width: 18.8vw; height: 1.9vw; margin: 3.9vw 0vw 0vw 4.2vw;" type = "text" name = "username" placeholder="Username..."/>

```

Comment:
I have updated the margins and style elements for input fields within this area.